### DISCUSSION TITLE
Problem Install on a Macbook (2016) HD could not be found

### DISCUSSION THREAD

**Discussion by Roadblaster66** [elementary/triage#289] [Q&A]
Even after a clean install of MacOS Monterey on the Macbook (early 2016), the installation of Elementery OS did not work. When choosing the hard disk, no HD was found, the HD was also not found via the option HD partition layout. Tried several times to no avail. Checked whether the installation version would be correct, installation on an older MacBook Pro 2009, the installation of Elementery could be performed, but the WIFI does not work there, so uninstalled there.

**Comment by lenemter** [comment]
I think its the same issue as https://github.com/elementary/installer/issues/617 which has been fixed and will be shipped very soon in OS 7.1.
